quickwit-core/src/indexing/split.rs

```diff
@@ -329,6 +330,10 @@ async fn put_split_files_to_storage(
     Ok(manifest)
 }
 
+/// [`FileEntry`] is an alias of [`ManifestEntry`] for
+/// holding the full path & size of a file.
+pub type FileEntry = ManifestEntry;
```

Comment:
Why not renaming `ManifestEntry` with `FileEntry` which is definitely better?

Reply:
It's not the same purpose, though the type structure is the same. 
- ManifestEntry is used in manifest and hold a relative path
- FileEntry is used for the user interface and holds the full path
I could be wrong but that was my thought process.

Reply:
Make sense but it's hard to guess this difference when looking at ManifestEntry struct. For me, it means that we can just rename it.
Generally speaking, I don't really like aliasing because it can make me believe I'm manipulating different things but it's not the case.